pwndbg/gdblib/heap/ptmalloc.py

```diff
@@ -758,7 +758,8 @@ def fastbins(self) -> Bins:
         fd_offset = pwndbg.gdblib.arch.ptrsize * 2
         safe_lnk = pwndbg.glibc.check_safe_linking()
         result = Bins(BinType.FAST)
-        for i in range(7):
+        num_fastbins = 7
+        for i in range(num_fastbins):
```

Comment:
I believe we don't need that change, just a comment is probably sufficient

Reply:
Perhaps use the NFASTBINS constant